manifests/default_mods.pp

```diff
@@ -14,19 +13,13 @@
   case $facts['os']['family'] {
     'redhat': {
       ::apache::mod { 'log_config': }
-      if versioncmp($apache_version, '2.4') >= 0 {
-        # Lets fork it
-        # Do not try to load mod_systemd on RHEL/CentOS 6 SCL.
-        if ( !($facts['os']['family'] == 'redhat' and versioncmp($facts['os']['release']['major'], '7') == -1) and !($facts['os']['name'] == 'Amazon') ) {
-          if ($use_systemd) {
-            ::apache::mod { 'systemd': }
-          }
-        }
-        if ($facts['os']['name'] == 'Amazon' and $facts['os']['release']['full'] == '2') {
-          ::apache::mod { 'systemd': }
-        }
-        ::apache::mod { 'unixd': }
+      if $facts['os']['name'] != 'Amazon' && $use_systemd {
+        ::apache::mod { 'systemd': }
       }
+      if ($facts['os']['name'] == 'Amazon' and $facts['os']['release']['full'] == '2') {
+        ::apache::mod { 'systemd': }
+      }
```

Comment:
Here I wonder what the best approach is. Why doesn't Amazon Linux support the `use_systemd` flag? And I think Amazon Linux 2022 is also out, which also has systemd. Should we simplify? Related: in other places I dropped Amazon Linux 1 support since it's based on EL6.